crates/pool/src/validate/mod.rs

```diff
@@ -0,0 +1,28 @@
+//! Transaction validation logic.
+
+use reth_chainspec::ChainSpec;
+use reth_transaction_pool::{
+    validate::{EthTransactionValidator, EthTransactionValidatorBuilder},
+    BlobStore,
+};
+use std::sync::Arc;
+
+/// A wrapper around the Reth [`EthTransactionValidatorBuilder`].
+/// The produced Validator will reject EIP4844 transactions not supported by Kakarot at the moment.
+#[derive(Debug)]
+pub struct KakarotTransactionValidatorBuilder(EthTransactionValidatorBuilder);
```

Comment:
Is this used anywhere? I think it should be passed somewhere at some point right?

Reply:
Oups this is true, just forgot to rearrange. I have just created a Kakarot Transaction Validator which is obtained from this builder and which is then used in the general Validator.

Reply:
Did you push? I can't find the Kakarot Transaction Validator